ios/FluentUI/Table View/TableViewHeaderFooterView.swift

```diff
@@ -151,6 +151,14 @@ open class TableViewHeaderFooterView: UITableViewHeaderFooterView {
         }
     }
 
+    /// If set, the accessory button title color will be based on this image, overiding the acessory style's title color.
+    /// The image will be adjusted to the button size, so the entire image based color is evenly distributed.
+     @objc open var imageForAccessoryButtonColor: UIImage? {
```

Comment:
I am not sure this is the direction we want to go for the continuous support of this control. shouldn't this be something like the TableViewCell which we expose the custom accessory button and client can do whatever they want with it? I think it will make it easier for like zero query search scenarios of client want to show ">"  button etc .... this image for accessory button color seems too specific

Reply:
it's a good point. It'll take me a a bit extra work and refactoring, so I'll revert this and try to make it a separate tentative change, since it's not that important for the Pride update.